tabbycat/utils/tables.py

```diff
@@ -341,6 +341,21 @@ def _result_cell_class_four_elim(self, advancing, cell):
             cell['sort'] = 0
         return cell
 
+    def _show_ballots(self, show_ballots: bool, cell, reverse_tournament, ts):
```

Comment:
`reverse_tournament` is a function with a file-scope. No need to have it as an argument.